doc/nrf/releases_and_maturity/migration/migration_guide_spm_to_tf-m.rst

```diff
@@ -0,0 +1,37 @@
+.. _spm_to_tfm_migration:
+
+Migrating from Secure Partition Manager to Trusted Firmware-M
+#############################################################
+
+The Nordic Secure Partition Manager (SPM) was replaced with Trusted Firmware-M (TF-M) as the default trusted execution solution in the nRF Connect SDK 2.1.0.
+This change was made to enhance the security features of the SDK by integrating the more widely adopted TF-M, which aligns with the Arm Platform Security Architecture (PSA).
+
+The migration from SPM to TF-M requires changes in the application code and the partition configuration file.
```

Comment:
I'll remove the reference to `file` because the partition configuration might also be the one that is generated automatically. So it is either static, or generated, but anyway it is not the same as before SPM.